src/weights_and_measures.php

```diff
@@ -42,6 +42,12 @@
 const DATETIME_FILENAME        = "Ymd\THis";
 const DATETIME_FILENAMEZ       = "Ymd\THisO";
 
+/**
+ * UTF8 Helpfulness
+ */
+
+ const UTF8_BOM = "\xef\xbb\xbf";
```

Comment:
ooooohhh maybe toss the other bad boys in here while we're up to it?

```suggestion
/**
 * Unicode Helpfulness
 */

const UTF8_BOM = "\xef\xbb\xbf";
const UTF16BE_BOM = "\xfe\xff";
const UTF16LE_BOM = "\xff\xfe";
const UTF32BE_BOM = "\0\0\xfe\xff";
const UTF32LE_BOM = "\xff\xfe\0\0"
```

Reply:
🔥